tests/apps/search/test_query_courses_edge_cases.py

```diff
@@ -181,39 +226,53 @@ def test_query_courses_rare_facet_force(self, *_):
         # - at the end for the one that is not in the top 10 and should not be included
         #   if it wasn't forced...
         response = self.client.get(
-            "/api/v1.0/courses/?organizations=L-00030002&organizations=L-00030011"
+            "/api/v1.0/courses/?organizations=L-00030002&organizations=L-0003000B"
         )
         self.assertEqual(response.status_code, 200)
         content = json.loads(response.content)
 
         self.assertEqual(
             content["filters"]["organizations"]["values"],
             [
-                {"count": 2, "human_name": "#L-00030001", "key": "L-00030001"},
-                {"count": 2, "human_name": "#L-00030002", "key": "L-00030002"},
-                {"count": 2, "human_name": "#L-00030003", "key": "L-00030003"},
-                {"count": 2, "human_name": "#L-00030004", "key": "L-00030004"},
-                {"count": 2, "human_name": "#L-00030005", "key": "L-00030005"},
-                {"count": 2, "human_name": "#L-00030006", "key": "L-00030006"},
-                {"count": 2, "human_name": "#L-00030007", "key": "L-00030007"},
-                {"count": 2, "human_name": "#L-00030008", "key": "L-00030008"},
-                {"count": 2, "human_name": "#L-00030009", "key": "L-00030009"},
-                {"count": 2, "human_name": "#L-00030010", "key": "L-00030010"},
-                {"count": 1, "human_name": "#L-00030011", "key": "L-00030011"},
+                {"count": 2, "human_name": "Organization #0", "key": "L-00030001"},
+                {"count": 2, "human_name": "Organization #1", "key": "L-00030002"},
+                {"count": 2, "human_name": "Organization #2", "key": "L-00030003"},
+                {"count": 2, "human_name": "Organization #3", "key": "L-00030004"},
+                {"count": 2, "human_name": "Organization #4", "key": "L-00030005"},
+                {"count": 2, "human_name": "Organization #5", "key": "L-00030006"},
+                {"count": 2, "human_name": "Organization #6", "key": "L-00030007"},
+                {"count": 2, "human_name": "Organization #7", "key": "L-00030008"},
+                {"count": 2, "human_name": "Organization #8", "key": "L-00030009"},
+                {"count": 2, "human_name": "Organization #9", "key": "L-0003000A"},
+                {"count": 1, "human_name": "Organization #10", "key": "L-0003000B"},
```

Comment:
I'm curious what made it change to hexadecimal?

Reply:
Node paths are always hexadecimal. It's just when we wrote them manually we wrote them in decimal 😅

Reply:
ah